README.md

```diff
@@ -1,181 +1,146 @@
 
-Check the Base64-encoded checkbox, click Generate New Credentials, and copy the generated credentials:
-![Generate Credentials](/assets/credentials.png)
+### Add new GitHub Actions Secrets 
 
-In your GitHub repo, go to Settings -> Secrets -> Actions and click New repository secret. Create a new secret named "MONDOO_CLIENT_ACCOUNT" with the contents you copied from the Mondoo Console:
-![Generate Credentials](/assets/secret.png)
-
-## Workflow Configuration Options
-
-The Mondoo GitHub Action has three required `with` values that must be set in your workflow configuration file:
-
-**service_account_credentials**
-
-The Mondoo service account credentials. Store these in a GitHub secret. Do not set them directly in the workflow configuration file. Once a secret is set up (as shown above) you can reference that secret in your workflow configuration file as `${{ secrets.MONDOO_CLIENT_ACCOUNT }}`.
-
-**scan_type**
-
-The type of Mondoo scan to perform:
-
-- `k8s` for Kubernetes manifest scanning.
-- `terraform` for Terraform configuration file scanning.
-- `docker_image` for scanning of Docker images from a Docker registry or from earlier GitHub actions steps.
-- `docker_image_from_dockerfile` for scanning of Docker images from a Dockerfile. Note: This will build and then scan the image which make be a lengthy process.
-
-**path**
-
-The file to scan with Mondoo or, if `scan_type` is set to `docker_image_from_dockerfile`, the path to the Dockerfile. Examples: `nginx.yml` or `Dockerfile`
-
-**docker_image_name**
-
-The container image name to scan when `scan_type` is set to `docker_image`. ex: `nginx:22.04`
-
-**score_threshold**
-
-The score threshold for scans. Value can be any number from `0-100`. If any score falls below the threshold, exit 1. Default value is `0`.
-
-**output_format**
-
-Set the output format (`compact`|`full`|`csv`|`json`|`junit`|`yaml`). Default `compact`.
-
-**extra_args**
-
-Allows specify extra command-line arguments for the Mondoo scan command.
+1. Click on **Settings** in your GitHub repository.
+2. Under the **Security** section click on **Actions**.
+3. Click **New repository secret**.
+4. Name the secret `MONDOO_SERVICE_ACCOUNT` and paste the base64 encoded credentials from the previous section into the value input.
+5. Click **Add secret**.
 
 ## Examples Workflows
 
 Simple scan of nginx.yml Kubernetes manifest:
 
 ```yaml
-name: mondoo-scan
-
+name: Mondoo Kubernetes Manifest scan
 on:
-  pull_request:
   push:
-    branches: [main]
-
+    paths:
+    - 'k8s/*.yaml'
 jobs:
-  build:
+  install:
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v3
-      - name: Scan with Mondoo
-        uses: mondoohq/actions@main
-        with:
-          service_account_credentials: ${{ secrets.MONDOO_CLIENT_ACCOUNT }}
-          scan_type: k8s
-          path: nginx.yml
+    - uses: actions/checkout@master
+    - uses: mondoohq/actions/kubernetes@master
```

Comment:
I already had the painful experience of having my workflows break "randomly" because I followed the instructions in our readme and they told me to use `master` as version. Can we start using semver or something that allows users to use a stable channel that has no surprising breaking changes?

This whole PR is a breaking change on its own. I think we should have v1, v2, etc. at least if not full semver

Reply:
@imilchev great point! I completely agree that this is a breaking change. I think there are to things to address here:

1. versioning - I feel like this is a major release, is it not? If so, I feel like we need to merge and then cut a release with the new major version

2. examples - once the new release is up, we can update the examples. I am not clear on what the best strategy is for the examples. Every other action I've looked at just uses `@master` in their examples.

Reply:
The issue with examples and versions is we would need to constantly update them. Otherwise users are going to copy the example and leave in an old/unsupported version pin.

Reply:
Don't get me wrong, I am not saying we should pin to a specific version. I think we should have the full semver but also have `v1`, `v2`, etc. I think the examples should always have the latest major version in them. I don't expect we will be making breaking changes every month so we won't need to update our READMEs that often

Reply:
another thing we thing to think about (not necessarily now) is how do we manage the Mondoo Client releases in the actions. Right now we always install the latest Mondoo Client for any action release. This can cause issues in the future since our action can be broken indirectly by the new release of an incompatible Mondoo Client. We should do some version pinning of the client like we do in our operator

Reply:
@imilchev I have been talking to @chris-rock about this. He is going to update Mondoo Client so that the config can be configured completely via an ENV VAR. Once that happens, I will update the action to use the container builds we push with each release. The examples will pull `latest` by default, but will be configurable and we can add documentation to pin the version. Thoughts?

Reply:
@imilchev @tas50 we are in agreement that we will come up with a strategy for version pinning once the new functionality @chris-rock is working on has been released. 

I have opened https://github.com/mondoohq/actions/issues/21 for tracking